server/athenian/api/openapi/openapi.yaml

```diff
@@ -860,12 +860,6 @@ paths:
               schema:
                 $ref: '#/components/schemas/InvalidRequestError'
           description: The request was invalid.
-        403:
```

Comment:
Can any user create a team?

Reply:
yup as described in [ENG-726](https://athenianco.atlassian.net/browse/ENG-726):

> Adding a new team should take place in the settings where a dedicated page allows all users (both regular and admin) to manage the teams associated with the corresponding Athenian account.